mpp/include/mpp_comm_mpi.inc

```diff
@@ -110,11 +110,11 @@
      debug   = flags.EQ.MPP_DEBUG
      verbose = flags.EQ.MPP_VERBOSE .OR. debug
   end if
-  if (t_level == 3) return
+  if (t_level == mpp_init_test_datatype_list_init) return
 
   call mpp_init_logfile()
   call read_input_nml
-  if (t_level == 4) return
+  if (t_level == mpp_init_test_logfile_init) return
```

Comment:
It is unwise to insert a bunch of lines into your production code simply to support testing. I hope this is not going to become a habit, or you will unnecessarily increase your costs and defect count.